### PR TITLE
Miscellaneous updates to docs

### PR DESCRIPTION
**Pull request by susodapop**
Added details about Query Filters and a cross-reference for dashboard-level filters. Added SSH Tunnel information (useful for Algolia search primarily). Wrote about Redash's dashboard refresh behavior..

### CODE REVIEW

File: src/pages/kb/data-sources/querying-elasticsearch.md [hunk 1]
```diff
@@ -70,4 +70,6 @@ source of type `Elasticsearch`.
 }
 ```
     
+## A note on authentication
 
+Redash has two data sources for ElasticSearch: standard and Amazon ElasticSearch service. In general, you should use the Amazon data source if you host your ElasticSearch instance there. However, if your Amazon ES instance does not require authentication then you **must** use the standard ES data source. The Amazon ES Data source only works when the underlying ES instance requires authentication.
```
Comment (arikfr) [inline src/pages/kb/data-sources/querying-elasticsearch.md]: 1. `ElasticSearch` -> `Elasticsearch`.
2. The instructions are a bit confusing. I would just rephrase it to: you need to use the Amazon ES type if you're using IAM based authentication, otherwise use the regular one.

File: src/pages/kb/user-guide/dashboards/dashboard-editing.md [hunk 2]
```diff
@@ -63,3 +63,18 @@ Now the Dashboard options menu includes a `Manage Permissions` option. Clicking
 
 Please note that currently the users you add won't receive a notification, so you will need to notify them manually.
 
+## Dashboard Refresh
+
+When you open a dashboard, Redash displays the most recent cached query result for each visualization. Dashboard data may grow stale as a result, depending on how often you run the underlying queries (either manually or with a scheduled query). However, you can force Redash to execute all of the underlying queries by clicking the Refresh button on the upper-right.
```
Comment (arikfr) [inline src/pages/kb/user-guide/dashboards/dashboard-editing.md]: Maybe link `scheduled query` to the relevant section?

File: src/pages/kb/faq/hosted-redash_category.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Each time you run a query we cache the query results.  This means there's
 always some subset of your data cached in our system. The most recent query
 result is cached indefinitely.
 
-## How to Cancel My Subscription
+## How to Cancel My Subscription?
```
Comment (ranbena) [inline src/pages/kb/faq/hosted-redash_category.md]:
```suggestion
## How Can I Cancel My Subscription?
```
Or else the question mark seems wrong

File: src/pages/kb/user-guide/dashboards/dashboard-editing.md [hunk 4]
```diff
@@ -63,3 +63,18 @@ Now the Dashboard options menu includes a `Manage Permissions` option. Clicking
...
+
+When you open a dashboard, Redash displays the most recent cached query result for each visualization. Dashboard data may grow stale as a result, depending on how often you run the underlying queries (either manually or with a scheduled query). However, you can force Redash to execute all of the underlying queries by clicking the Refresh button on the upper-right. 
+
+If you want your dashboards to stay fresh but don't want to make scheduled queries, you can use Automatic Dashboard Refresh. When enabled, the underlying queries will be executed periodically as long as your browser is open.
+
+![](/assets/images/docs/gitbook/dashboard-refresh.png)
+
+{% callout info %}
+
+Automatic Dashboard Refresh occurs as part of the Redash frontend application. Your refresh schedule is only in-effect as long as a logged-in user has the dashboard open in their browser. To guarantee that your queries are executed regularly (which is important for alerts), you should use a Scheduled Query.
```
Comment (ranbena) [inline src/pages/kb/user-guide/dashboards/dashboard-editing.md]:
```suggestion
Automatic Dashboard Refresh occurs as part of the Redash frontend application. Your refresh schedule is only in-effect as long as a logged-in user has the dashboard open in their browser. To guarantee that your queries are executed regularly (which is important for alerts), you should use a [Scheduled Query]({% link _kb/user-guide/querying/scheduling-a-query.md  %}) instead.
```

File: src/pages/kb/user-guide/dashboards/dashboard-editing.md [hunk 5]
```diff
@@ -63,3 +63,18 @@ Now the Dashboard options menu includes a `Manage Permissions` option. Clicking
...
+
+{% endcallout %}
+
+Importantly, when visitors access a public dashboard they will always see the cached query result. Refreshing a public dashboard does **not** execute the underlying queries. The only way that visitors to a public dashboard link will see refreshed data is if a logged-in Redash user executes the underlying queries or if a query schedule is in effect.
```
Comment (ranbena) [inline src/pages/kb/user-guide/dashboards/dashboard-editing.md]: @arikfr is the reasoning behind this to prevent untraceable abuse?

File: src/pages/kb/data-sources/querying-elasticsearch.md (hunk 1 above)
Comment (susodapop) [reply src/pages/kb/data-sources/querying-elasticsearch.md]: Corrected. Thanks!

File: src/pages/kb/user-guide/dashboards/dashboard-editing.md (hunk 2 above)
Comment (susodapop) [reply src/pages/kb/user-guide/dashboards/dashboard-editing.md]: @ranbena handled this.

File: src/pages/kb/user-guide/dashboards/dashboard-editing.md (hunk 5 above)
Comment (susodapop) [reply src/pages/kb/user-guide/dashboards/dashboard-editing.md]: Anything holding this back from merge now?

**Comment by ranbena** [comment]
I would reckon for help docs, one round of feedback should do and unless there's an open discussion, should always be good to merge.

**Comment by arikfr** [comment]
👍
